MockMDS/src/main/java/io/mosip/mock/sbi/service/SBIMockService.java

```diff
@@ -225,7 +226,7 @@ private static synchronized boolean isPortInUse (int port)
 		}
 		catch (Exception ex)
 		{
-			LOGGER.error("Socket Not available {}" , port , ex);
+			LOGGER.info("Socket Not available {}" , port , ex);

```

Comment:
why info? this is an error log

Reply:
This is not error in our case, this is loop we will be getting another port in sometime.

Reply:
Changed to info